### ISSUE TITLE
Gulp default task takes a very long time.

### ISSUE THREAD

**Issue by Yugloocamai** [magicien/JSceneKit#2]
Why is that?  Took about 30 seconds on my i7 Macbook.  I would suggest at least adding gulp-debug so we can get some verbosity.  I thought it was broken at first.

**Comment by magicien** [comment]
You're right. It used to show a lot of logs, and I thought it was messy and made it silent. But it should show the progress.
I'm thinking of updating webpack to v4. It might shorten the compile time.
